addons/hr_calendar/tests/common.py

```diff
@@ -0,0 +1,123 @@
+                (0, 0, {'name': 'Tuesday Evening', 'dayofweek': '1', 'hour_from': 13, 'hour_to': 16, 'day_period': 'afternoon'}),
+                (0, 0, {'name': 'Wednesday Morning', 'dayofweek': '2', 'hour_from': 8, 'hour_to': 12, 'day_period': 'morning'}),
+                (0, 0, {'name': 'Wednesday Lunch', 'dayofweek': '2', 'hour_from': 12, 'hour_to': 13, 'day_period': 'lunch'}),
+                (0, 0, {'name': 'Wednesday Evening', 'dayofweek': '2', 'hour_from': 13, 'hour_to': 16, 'day_period': 'afternoon'}),
+                (0, 0, {'name': 'Thursday Morning', 'dayofweek': '3', 'hour_from': 8, 'hour_to': 12, 'day_period': 'morning'}),
+                (0, 0, {'name': 'Thursday Lunch', 'dayofweek': '3', 'hour_from': 12, 'hour_to': 13, 'day_period': 'lunch'}),
+                (0, 0, {'name': 'Thursday Evening', 'dayofweek': '3', 'hour_from': 13, 'hour_to': 16, 'day_period': 'afternoon'}),
+                (0, 0, {'name': 'Friday Morning', 'dayofweek': '4', 'hour_from': 8, 'hour_to': 12, 'day_period': 'morning'}),
+                (0, 0, {'name': 'Friday Lunch', 'dayofweek': '4', 'hour_from': 12, 'hour_to': 13, 'day_period': 'lunch'}),
+                (0, 0, {'name': 'Friday Evening', 'dayofweek': '4', 'hour_from': 13, 'hour_to': 16, 'day_period': 'afternoon'})
+            ],
+        })
+        cls.calendar_28h_company_B = cls.calendar_28h.copy()
+        cls.calendar_28h_company_B.company_id = cls.company_B
+
+        cls.calendar_35h_night = cls.env['resource.calendar'].create({
+            'company_id': cls.company_A.id,
+            'tz': "Europe/Brussels",
+            'name': 'night calendar',
+            'attendance_ids': [
+                (0, 0, {'name': 'Monday Evening', 'dayofweek': '0', 'hour_from': 15, 'hour_to': 22, 'day_period': 'afternoon'}),
+                (0, 0, {'name': 'Tuesday Evening', 'dayofweek': '1', 'hour_from': 15, 'hour_to': 22, 'day_period': 'afternoon'}),
+                (0, 0, {'name': 'Wednesday Evening', 'dayofweek': '2', 'hour_from': 15, 'hour_to': 22, 'day_period': 'afternoon'}),
+                (0, 0, {'name': 'Thursday Evening', 'dayofweek': '3', 'hour_from': 15, 'hour_to': 22, 'day_period': 'afternoon'}),
+                (0, 0, {'name': 'Friday Evening', 'dayofweek': '4', 'hour_from': 15, 'hour_to': 22, 'day_period': 'afternoon'})
+            ],
+        })
+
+        cls.calendar_35h_night_company_B = cls.calendar_35h_night.copy()
+        cls.calendar_35h_night_company_B.company_id = cls.company_B
+
+        cls.partnerA = cls.env['res.partner'].create({
+            'name': "Partner A"
+        })
+        cls.partnerB = cls.env['res.partner'].create({
+            'name': "Partner B"
+        })
+        cls.partnerC = cls.env['res.partner'].create({
+            'name': "Partner C"
+        })
+        cls.partnerD = cls.env['res.partner'].create({
+            'name': "Partner D"
+        })
```

Comment:
I think you can do something along those lines, same for above and below:
(+ nitpicking on quotes consistency)
```suggestion
        cls.partnerA, cls.partnerB, cls.partnerC, cls.partnerD = cls.env['res.partner'].create(
            {
                'name': 'Partner A'
            },
            {
                'name': 'Partner B'
            },
            {
                'name': 'Partner C'
            },
            {
                'name': 'Partner D'
            },
        )
```